src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -51,7 +60,11 @@
 
 helps['backup item set-policy'] = """
 type: command
-short-summary: Update the policy associated with this item.
+short-summary: Update the policy associated with this item. Use this to change policies of the backup item.
+examples:
+  - name: Update the policy associated with this item. Use this to change policies of the backup item. (autogenerated)
```

Comment:
Why do you have "(autogenerated)" in your example name and with the property "crafted:true"? Typically it is used for Aladdin generated examples, are your examples auto-generated?